packages/tools/fluid-runner/src/fluidRunner.ts

```diff
@@ -6,50 +6,56 @@
 import * as fs from "fs";
 import * as yargs from "yargs";
 import { ChildLogger } from "@fluidframework/telemetry-utils";
-import { exportFile } from "./exportFile";
+import { exportFile, IExportFileResponse, parseBundleAndExportFile } from "./exportFile";
 // eslint-disable-next-line import/no-internal-modules
 import { FileLogger } from "./logger/FileLogger";
+import { IFluidFileConverter } from "./codeLoaderBundle";
 
-function fluidRunner() {
+/**
+ * @param fluidFileConverter - needs to be provided if "codeLoaderBundle" is not and vice versa
+ */
+export function fluidRunner(fluidFileConverter?: IFluidFileConverter) {
     // eslint-disable-next-line @typescript-eslint/no-unused-expressions
     yargs
         .strict()
         .version(false)
         .command(
             "exportFile",
-            "Generate an output for a local snapshot",
+            "Generate an output for a local ODSP snapshot",
             // eslint-disable-next-line @typescript-eslint/no-shadow
             (yargs) =>
                 yargs
                     .option("codeLoader", {
-                        describe: "Name of the code loader bundle",
+                        // eslint-disable-next-line max-len
+                        describe: "Path to code loader bundle. Required if this application is being called without modification.\nSee \"README.md\" for more details.",
                         type: "string",
-                        demandOption: true,
+                        demandOption: false,
```

Comment:
Is this optional? If so, what happens if it's not provided?

Reply:
If it is not provided and we don't pass in an `IFluidFileConverter` then the application errors and closes. This is noted in the README.md